tests/unit/config/test_project.py

```diff
@@ -350,7 +350,7 @@ def test_cycle(self):
 
         assert "Cycle detected" in str(exc.exception)
 
-    def test_query_comment_disabled(self):
```

Comment:
DbtProject itself doesn't have any business logic to set query_comment.append to False if the query-comment is `None` or `""`. It is entirely driven by the QueryComment() contract which is now defined in dbt-adapters. So this test was really just testing empty comment values.